um_canonicalize/tags.yaml

```diff
@@ -308,12 +351,21 @@ categories:
   Tense:
     - PRS
     - PST
+    - PST.FREQ
+    - INF
     - FUT
     - IMMED
     - HOD
     - 1DAY
     - RCT
     - RMT
+    - PST+RMT
+    - FUT+RMT
+    - PST+RCT
+    - PST+IMMED
+    - PRS+IMMED
+    - FUT+IMMED
+    - NFUT
```

Comment:
"non-future" ?

Reply:
Yeah. `NFUT` → `non{FUT}`. This fix is actually highlighted on page 1 of the 3.0 paper.

Reply:
Ok. We need to add non{FUT} to yaml then.